presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -306,6 +306,20 @@ private AutoCloseable withTable(String tableName, String tableDefinition)
         };
     }
 
+    @Test
+    public void testCreateSchema()
```

Comment:
Add the test to `AbstractTestIntegrationSmokeTest` since the functionality is provided by base-jdbc, not postgres connector specific.

Reply:
@Praveen2112 can you please address this comment?

Reply:
While I agree with adding a test to the class, we still need to override it in each jdbc connector (because DROP SCHEMA implementation was skipped in this PR).

Reply:
That's ok.